engine/buff/sc_buff.cpp

```diff
@@ -2390,7 +2397,14 @@ void buff_t::aura_loss()
   {
     if ( !player->is_sleeping() )
     {
-      sim->print_log( "{} loses {}", *player, name_str );
+      if ( sim->log_spell_id )
+      {
+        sim->print_log( "{} loses {} ({})", *player, buff_display_name, s_data->id() );
```

Comment:
```suggestion
        sim->print_log( "{} loses {} ({})", *player, name_str, s_data->id() );
```
This should be name_str to mirror the display without log_spell_id